app/code/Magento/CheckoutAgreements/Block/Adminhtml/Agreement/Edit/Form.php

```diff
@@ -32,10 +50,14 @@ public function __construct(
         \Magento\Framework\Data\FormFactory $formFactory,
         \Magento\Store\Model\System\Store $systemStore,
         \Magento\CheckoutAgreements\Model\AgreementModeOptions $agreementModeOptions,
+        \Magento\Cms\Model\Wysiwyg\Config $wysiwygConfig,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
```

Comment:
To maintain backward compatibility, please add new dependencies to the end of the list as optional parameters. See "Adding a constructor parameter" section of [Backward Compatibility Policy](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#php) for more details.